app/Traits/WmNovaFieldsTrait.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace App\Traits;
+
+use DKulyk\Nova\Tabs;
+use App\Models\UgcPoi;
+use App\Models\UgcTrack;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\Schema;
+
+trait WmNovaFieldsTrait
+{
+    /**
+     * Generate Nova fields based on a JSON schema or a provided schema array.
+     *
+     * @param string|null $name The name of the column where the form JSON is stored. Default is null.
+     * @param array|null $formSchema The optional schema for fields.
+     * @return array
+     * @throws \Exception
+     */
+    public function jsonForm(string $columnName, array $formSchema = null)
+    {
+        // Ensure Laravel Nova is installed
+        $this->ensureNovaIsInstalled();
+
+        if (!isset($this->attributes) || !array_key_exists($columnName, $this->attributes)) {
+            return [];
+        }
+
+        if (strpos($this->app_id, 'osm2cai') !== false) {
+            return [];
+        }
+
+        $appId = strpos($this->app_id, 'geohub_') !== false
+            ? substr($this->app_id, strpos($this->app_id, 'geohub_') + strlen('geohub_'))
+            : $this->app_id;
+
+        $geohubAppConfig = 'https://geohub.webmapp.it/api/app/webmapp/' . $appId . '/config.json';
+        $config = Cache::remember('geohub_app_config_' . $appId, now()->addHour(), function () use ($geohubAppConfig) {
+            $response = Http::get($geohubAppConfig);
+            return $response->json();
+        });
+        $acquisitionForm = $this->getAcquisitionForm($config);
+
+        $fields = [];
+        foreach ($acquisitionForm as $formSection) {
+            if ($this->form_id != $formSection['id']) {
+                continue;
+            }
+            $tabsLabel = $formSection['label']['it'] ?? $formSection['label']['ït'] ?? $formSection['label']['en'];
```

Comment:
creare una funzione che embeddi questo codice
$tabsLabel = $formSection['label']['it'] ?? $formSection['label']['ït'] ?? $formSection['label']['en'];